xdem/coreg/affine.py

```diff
@@ -848,6 +788,34 @@ def to_matrix(self) -> NDArrayf:
         """Convert the transform to a 4x4 transformation matrix."""
         return self._to_matrix_func()
 
+    def to_translations(self) -> tuple[float, float, float]:
+        """
+        Convert the affine transformation matrix to only its X/Y/Z translations.
```

Comment:
I think something funky will happen here if scaling is involved (I don't remember which order the components are applied in). I feel like a warning should be added to this and the `to_rotations()` saying something like "Warning: Information may be lost".

And the conversion is less of a conversion and more of an extraction I would say! Since there's no validation that scaling won't mess with the result.

Reply:
I agree that it's not a conversion, the idea was just to have an extraction! We could change this in the description? 
Leaving this conversation open for @adehecq to comment.

Reply:
And I'll look into it for when scaling is involved, great point :wink:

Reply:
For the record: translation is OK with scaling, but the rotation is affected. Opening an issue to add this in the future.